src/stores/mod.rs

```diff
@@ -0,0 +1,233 @@
+use std::cmp::Eq;
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
+use std::hash::Hash;
+
+use super::Cached;
+
+#[cfg(feature = "async")]
+use {super::CachedAsync, async_trait::async_trait, futures::Future};
+
+pub mod sized;
+pub mod timed;
+pub mod timed_sized;
+pub mod unbound;
+
+pub type UnboundCache<K, V> = unbound::UnboundCache<K, V>;
```

Comment:
do these need to be defined as `pub type` or could they be a `pub use unbound::UnboundCache`, etc?

Reply:
Done. Also `pub mod unbound;` and all others should not be public!